backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,72 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: create-upload-table
+      author: dsass@skylight.digital
+      comment: Add a table that holds data about bulk test result uploads.
+      changes:
+        - tagDatabase:
+            tag: create-upload-table
+        - sql:
+            remarks: Create the enumerations for upload statuses
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.UPLOAD_STATUS as ENUM('SUCCESS', 'FAILURE', 'PENDING');
+        - createTable:
+            tableName: upload
+            remarks: A collection of meta data for bulk uploads.
+            columns:
+              - column:
+                  name: report_id
+                  type: uuid
+                  remarks: The Report Stream report Id.
+                  constraints:
+                    nullable: false
+              - column: *pk_column
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
+              - column:
+                  name: records_count
+                  remarks: The number of records uploaded.
+                  type: int
+                  constraints:
+                    nullable: false
+              - column:
+                  name: status
+                  remarks: The status of the upload.
+                  type: ${database.defaultSchemaName}.UPLOAD_STATUS
+                  constraints:
+                    nullable: false
+              - column:
+                  name: org_id
+                  remarks: The organization id that uploaded the data.
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__upload__organization
+                    references: organization(internal_id)
+              - column:
+                  name: facility_id
+                  remarks: The facility id that uploaded the data.
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__upload__facility
+                    references: facility(internal_id)
+              - column:
+                  name: errors
+                  remarks: Any errors encountered when uploading.
+                  type: text
```

Comment:
jsonb? probably not necessary?

Reply:
We do use jsonb elsewhere in the database, I don't see a problem using it here as well.

Reply:
I think it makes sense to opt out of jsonb because we don't have control over these values.  And if report stream changes their api for some reason I'd rather that not break things.

Reply:
Makes sense to me! Happy to keep it as `text` then.